tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -6460,20 +6473,33 @@ TEST_F(SQLiteFixture, WriteVeriSumTable_TestNotPartOfTotal)
 
         {state->dataHeatBal->Zone(1).Name,
          "Plug and Process",
-         state->dataHeatBal->ZoneElectric(1).DesignLevel / state->dataHeatBal->Zone(1).FloorArea},
+         (state->dataHeatBal->ZoneElectric(1).DesignLevel + state->dataHeatBal->ZoneITEq(1).DesignTotalPower) /
+             state->dataHeatBal->Zone(1).FloorArea},
         {state->dataHeatBal->Zone(2).Name,
          "Plug and Process",
-         state->dataHeatBal->ZoneElectric(2).DesignLevel / state->dataHeatBal->Zone(2).FloorArea},
+         (state->dataHeatBal->ZoneElectric(2).DesignLevel + state->dataHeatBal->ZoneITEq(2).DesignTotalPower) /
+             state->dataHeatBal->Zone(2).FloorArea},
         {state->dataHeatBal->Zone(3).Name,
          "Plug and Process",
-         state->dataHeatBal->ZoneElectric(3).DesignLevel / state->dataHeatBal->Zone(3).FloorArea},
+         (state->dataHeatBal->ZoneElectric(3).DesignLevel + state->dataHeatBal->ZoneITEq(3).DesignTotalPower) /
+             state->dataHeatBal->Zone(3).FloorArea},
         {"Total",
          "Plug and Process",
-         (state->dataHeatBal->ZoneElectric(1).DesignLevel + state->dataHeatBal->ZoneElectric(2).DesignLevel) /
+         (state->dataHeatBal->ZoneElectric(1).DesignLevel + state->dataHeatBal->ZoneElectric(2).DesignLevel +
+          +state->dataHeatBal->ZoneITEq(1).DesignTotalPower + state->dataHeatBal->ZoneITEq(2).DesignTotalPower) /
              (state->dataHeatBal->Zone(1).FloorArea + state->dataHeatBal->Zone(2).FloorArea)},
-        {"Conditioned Total", "Plug and Process", state->dataHeatBal->ZoneElectric(1).DesignLevel / state->dataHeatBal->Zone(1).FloorArea},
-        {"Unconditioned Total", "Plug and Process", state->dataHeatBal->ZoneElectric(2).DesignLevel / state->dataHeatBal->Zone(2).FloorArea},
-        {"Not Part of Total", "Plug and Process", state->dataHeatBal->ZoneElectric(3).DesignLevel / state->dataHeatBal->Zone(3).FloorArea},
+        {"Conditioned Total",
+         "Plug and Process",
+         (state->dataHeatBal->ZoneElectric(1).DesignLevel + state->dataHeatBal->ZoneITEq(1).DesignTotalPower) /
+             state->dataHeatBal->Zone(1).FloorArea},
+        {"Unconditioned Total",
+         "Plug and Process",
+         (state->dataHeatBal->ZoneElectric(2).DesignLevel + state->dataHeatBal->ZoneITEq(2).DesignTotalPower) /
+             state->dataHeatBal->Zone(2).FloorArea},
+        {"Not Part of Total",
+         "Plug and Process",
+         (state->dataHeatBal->ZoneElectric(3).DesignLevel + state->dataHeatBal->ZoneITEq(3).DesignTotalPower) /
+             state->dataHeatBal->Zone(3).FloorArea},
```

Comment:
Update existing test to make sure that ITE power is included in the Zone Summary report.

Reply:
This is fine, but it's pretty hard to read.  Next time, please do a quick cleanup replacing all the `state->dataHeatBal->ZoneElectric(1)` with `zElectric1` and similar to other long variable expressions.  Anyway, this is fine.